Description: Minute typesetting improvements.
 A missing line break, and some inconsistent
 use of boldface characters.
 .
 Write the names of executable in boldface when
 referencing to them in the section "See also".
Author: Mats Erik Andersson <debian@gisladisker.se>
Last-Update: 2011-04-14

Index: b/cmd/dot/dot.1
===================================================================
--- a/cmd/dot/dot.1
+++ b/cmd/dot/dot.1
@@ -152,16 +152,17 @@
 default attributes.
 .PP
 \fBn0 [\fIname0=val0,name1=val1,...\fB];\fR
+.br
 Creates node \fBn0\fP (if it does not already exist)
 and sets its attributes according to the optional list. 
 .PP
-\fBn0 \fIedgeop\fR n1 \fIedgeop\fR \fI...\fB \fIedgeop\fR nn [\fIname0=val0,name1=val1,...\fB];\fR
+\fBn0 \fIedgeop\fR n1 \fIedgeop\fR \fI...\fB \fIedgeop\fR nn \fB[\fIname0=val0,name1=val1,...\fB];\fR
 .br
 Creates edges between nodes \fBn0\fP, \fBn1\fP, ..., \fBnn\fP and sets
 their attributes according to the optional list.
 Creates nodes as necessary.
 .PP
-[\fBsubgraph \fIname\fB] { \fIstatement\(hylist \fB}\fR
+[\fBsubgraph \fIname\fR] \fB{ \fIstatement\(hylist \fB}\fR
 .br
 Creates a subgraph.  Subgraphs may be used in place
 of \fBn0\fP, ..., \fBnn\fP in the above statements to create edges.
@@ -675,13 +676,7 @@
 on\(hyline reference pages. Most of these documents are also available in the
 \fIdoc\fP and \fIdoc/info\fP subtrees in the source and binary distributions.
 .PP
-dotty(1)
-.br
-tcldot(n)
-.br
-xcolors(1)
-.br
-libgraph(3)
+.BR dotty (1), tcldot (n), xcolors (1), libgraph (3).
 .PP
 E. R. Gansner, S. C. North,  K. P. Vo, "DAG \(hy A Program to Draw Directed Graphs", Software \(hy Practice and Experience 17(1), 1988, pp. 1047\(hy1062.
 .br
